src/Native/Runtime/unix/unixasmmacros.inc

```diff
@@ -3,6 +3,8 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+#include <AsmOffsets.inc>         // generated by the build from AsmOffsets.cpp
```

Comment:
I don't think it belongs to the macros and not all files that include unixasmmacros.inc need the offsets. I would prefer including the asmoffsets.inc selectively in the .S files where needed.